src/app/DebugImages.tsx

```diff
@@ -30,7 +30,7 @@ const IconItem = ({ Component, fileName }: { Component: React.ElementType; fileN
 function DebugImages() {
   return (
     <LinearGradient
-      colors={[Colors.background, Colors.gray3]}
+      colors={[Colors.background, Colors.inactive]}
```

Comment:
this one is kind of not semantic but it's a debug screen so didn't want to think too much on it